src/test/java/seedu/address/model/person/LevelTest.java

```diff
@@ -72,5 +73,20 @@ public void equals() {
 
         // different values -> returns false
         assertFalse(level.equals(new Level("S2 EXPRESS")));
+
+        // case-insensitive ->  returns true
+        assertTrue(level.equals(new Level("s1 express")));
+
+        // Ignore multi-spacing ->  returns true
+        assertTrue(level.equals(new Level("s1   express")));
+    }
+
+    @Test
+    public void toStringTest() {
+        Level l = new Level("S1 EXPRESS");
+        Level l2 = new Level("s1 express");
+        String expected = "S1 EXPRESS";
+        assertEquals(l.toString(), expected);
+        assertEquals(l2.toString(), expected);
```

Comment:
should tests be added under isValidLevel also to check that it is true for lower case, with extra space, etc?